vnet/lib/vnet/models/ip_lease.rb

```diff
@@ -60,6 +60,11 @@ def where_network_id(network_id)
 
     end
 
+    def valid_in_subnet
+      ipv4_prefix = Network[network_id][:ipv4_prefix]
+      IPAddress::IPv4::parse_u32(Network[self.network_id][:ipv4_network], ipv4_prefix).include? IPAddress::IPv4::parse_u32(self.ipv4_address, ipv4_prefix)
```

Comment:
This line is too long. On one line you are parsing the network, parsing the IP address and checking if the IP is in the network. That's three statements on one line. Code is much easier to read if you do only one statement per line.

Also it is good practice to use `::` for separation between constants and `.` for a method call. Makes for a more clear distinction.

```ruby
network = IPAddress::IPv4.parse_u32(Network[self.network_id][:ipv4_network], ipv4_prefix)
ipv4 = IPAddress::IPv4.parse_u32(self.ipv4_address, ipv4_prefix)

network.include?(ipv4)
```